### PR TITLE
Fix/ppdsc 2208 Fix svg previewer: remove sourcepoint and tealium in dev 

### PR DESCRIPTION
**Pull request by Vanals**
PPDSC-2208

**What**

I have tried to remove the scripts only for the `tools/preview-illustration-export` page creating a flag base on the page but did not work.
I am then implementing the solution removing scripts on `dev`. We will be able to really test it one in there but hopefully this should work. Locally the scripts were not in the DOM

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2208-remove-scripts-forsvg-previer
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2208-remove-scripts-forsvg-previer/storybook

**Comment by Vanals** [comment]
@mutebg  also removed the e2e tests for the consent :/. I guess we can't keep them and we will be able to bring them back once the consent works on dev environment

**Review by mutebg — APPROVED** [review]

**Review by Srinivasan-Ramasamy — APPROVED** [review]

File: site/pages/_document.tsx [hunk 1]
```diff
@@ -35,14 +36,16 @@ export default class MyDocument extends Document {
...
           <HTMLMeta />
-          <Consent
-            sourcePointConfigUnified={{
-              accountId: 259,
-              propertyHref: 'https://newskit.co.uk',
-              gdpr: {},
-            }}
-            reactHelmet={Helmet}
-          />
+          {isSiteEnvProduction && (
```
Comment (jps) [inline site/pages/_document.tsx:39]: wouldn't it be better to use `useRouter` to do this? with this approach, we can just target the single page and not turn it off for the whole env build? 
https://nextjs.org/docs/api-reference/next/router

Comment (Vanals) [reply site/pages/_document.tsx:39]: I could not make useRouter work, I wonder if is usable in the _document.tsx? I tried useRouter and WithRouter. One for Functional components and the other for Classes, but keep getting an error which tells me I cannot use the hook if not in the body.
Happy to have a quick catch up and I can show you what I have tried so far. I agree would be better have a flag by page.

Comment (Vanals) [reply site/pages/_document.tsx:39]: @jps  I will make then a ticket for the backlog to put the tests back and investigate on the issue once dev is working.

**Pull request merged**